exasol_transformers_extension/upload_model.py

```diff
@@ -32,21 +34,37 @@ def upload_model(**kwargs) -> None:
     """
     Downloads model from Huggingface hub and the transfers model to database
     """
+    # create bucketfs location
+    bucketfs_location = create_bucketfs_location(**kwargs)
+
+    upload_model_to_bfs_location(kwargs[MODEL_NAME_ARG], kwargs[TASK_TYPE_ARG],
+                                 Path(kwargs[SUBDIR_ARG]), bucketfs_location, kwargs[TOKEN_ARG])
+
+
+def upload_model_to_bfs_location(model_name: str, task_type: str, subdir: Path,
+                                 bucketfs_location: PathLike, huggingface_token: Optional[str] = None) -> None:
+    """
```

Comment:
Can you return the model file path rather than print it? The print is not necessarily what the caller wants to happen.

Reply:
changed